boards/nxp/s32k5xxcvb/s32k5xxcvb_s32k566_r52_defconfig

```diff
@@ -0,0 +1,8 @@
+# Copyright 2025 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_ISR_STACK_SIZE=512
+CONFIG_SYS_CLOCK_TICKS_PER_SEC=1000
+CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC=4000000
```

Comment:
```suggestion
```
No.

Reply:
I've removed, also moved the `CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC` to SoC level